### ISSUE TITLE
Query Language link incorrect

### ISSUE THREAD

**Issue by andrelkin** [bastibe/annotate.el#79]
In the description `Query Language` is referred with an incorrect link, the correct one is https://github.com/bastibe/annotate.el#query-language.
Could someone please fix that?

**Comment by cage2** [comment]
Hi @andrelkin !

Thank you for your report, never noticed this issue! Seems to me that the link in the org-mode file was correct and perhaps the problem could be in the rendering engine of github. I just checked the source file and the generated HTML file from Emacs and in both formats the link works.

Any idea how can i forward this to github folks?

Bye!
C.

**Comment by cage2** [comment]
Hi!

I have contacted github about this, in the meanwhile maybe would be a good idea to remove the link from the README.

Bye!
C.

**Comment by andrelkin** [comment]
To remove looks good. Thank you for addressing it! I am really looking forward to make use
of your tool at my work place (and who knows if that won't be followed with contributions to the project :-)!).

**Comment by cage2** [comment]
Hi @andrelkin !

So the folks at github kindly answered my question and wrote tho change the internal link:

```
[[Query language]]
```

with 

```
[[https://github.com/bastibe/annotate.el#query-language][Query Language]]
```
This is exactly your first suggestion (and form their -i mean github- point of view a reasonable solution). :-D

But, instead, i feel this is a workaround that will break the ORG file so I am inclined to remove the link anyway. But i am happy to discuss using the first solution (changing, not removing, the link).

I think the right thing to do is to ask @bastibe (the maintainer)  for his opinion.

Anyway i hope that this package will be useful for your work (or fun :-) ) and i will be happy if you decided to contribute to this package! :)

The code, i think, is not too difficult to grasp and there are docstrings for each function and variable. There are some parts a bit convoluted so feel free to ask me if you need explaination, i'll try my best to answer your questions! :)

Bye!
C.

**Comment by bastibe** [comment]
I completely agree with you, @cage2, on all points. Github's org parser is broken, and I'd rather have no link than a link that works on Github, but not in Emacs. And annotate.el is a cool code base indeed, in no small part thanks to @cage2's contributions!

**Comment by cage2** [comment]
Hello everyone!

> I completely agree with you, @cage2, on all points. Github's org parser is broken, 

yes it is, i think they made a lot of improvements lately but i feel someway org is still a second class citizen here.

> and I'd rather have no link than a link that works on Github, but not in Emacs. 

OK! @andrelkin  if you agree i would remove the internal link.

> And annotate.el is a cool code base indeed, in no small part thanks to @cage2's contributions!

You are too kind, i am just playing with elisp! :-) :-) The code i wrote here would be a mess if not for your suggestions and constructive critics, i feel this is truly a collective efforts.

Bye!
C.

**Comment by cage2** [comment]
Fixed by #81
